tests/Client/UpdaterTest.php

```diff
@@ -6,6 +6,7 @@
 use PHPUnit\Framework\TestCase;
 use Tuf\Client\Updater;
 use Tuf\Exception\MetadataException;
+use Tuf\Exception\PotentialAttackException\RollbackAttackException;
```

Comment:
Out of scope, but I'd love to rename this namespace to remove the redundant use of the term "exception". IMHO It should be `Tuf\Exception\PotentialAttack`.